includes/class-api.php

```diff
@@ -322,15 +322,18 @@ public function sync_post( $post_args, $duplicate_action, $force_update = false
 		}
 
 		// Add categories.
-		if ( isset( $post_args['tax_input']['category'] ) && $post_args['tax_input']['category'] ) {
+		if ( ! empty( $post_args['tax_input']['category'] ) ) {
 
 			require_once( ABSPATH . '/wp-admin/includes/taxonomy.php' );
 
 			foreach ( $post_args['tax_input']['category'] as $category ) {
 				wp_insert_category( array(
-					'cat_name' => $category,
+					'cat_name'             => $category['name'],
+					'category_description' => strlen( $category['description'] ) ? $category['description'] : '',
```

Comment:
Curious about the rationale for the conditional here. Couldn't we just bring over the description whether or not it has a string?